config/katello.migrations/190314163941-remove-foreman-user-groups.rb

```diff
@@ -1,5 +1,3 @@
-if answers['foreman'].is_a?(Hash)
+if answers['foreman'].is_a?(Hash) && answers['foreman'].key?('user_groups')
   answers['foreman']['user_groups'] = []
-elsif answers['foreman']
-  answers['foreman'] = { 'user_groups' => [] }
```

Comment:
Looking at the history and timeline behind https://github.com/theforeman/foreman-installer/pull/331/commits/65039e54e1973c2893e900c62ba78e23b42132ad , I think it should be removed since you are also removing `user_groups: []` from the default answers

Reply:
Indeed, it should be taken care of by the new dynamic-puppet-in-foreman-groups. That said, I think we should remove the file entirely rather than keeping a dummy.